packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerPlugin.m

```diff
@@ -3,60 +3,20 @@
 // found in the LICENSE file.
 
 #import "FVPVideoPlayerPlugin.h"
-#import "FVPVideoPlayerPlugin_Test.h"
 
 #import <AVFoundation/AVFoundation.h>
-#import <GLKit/GLKit.h>
 
-#import "./include/video_player_avfoundation/AVAssetTrackUtils.h"
+#import "./include/video_player_avfoundation/FVPAVFactory.h"
 #import "./include/video_player_avfoundation/FVPDisplayLink.h"
+#import "./include/video_player_avfoundation/FVPFrameUpdater.h"
+#import "./include/video_player_avfoundation/FVPVideoPlayerPlugin_Test.h"
+#import "./include/video_player_avfoundation/FVPVideoPlayer_Test.h"
```

Comment:
A non-test file should never import another file's `_Test.h` header. If something here actually needs this import, that should be in the main header instead of the test header.

Reply:
Right, then `FlutterStreamHandler` and `FlutterTexture` conformance had to be moved to the main header file now